tests/acceptance/features/apiGraph/userGDPRExport.feature

```diff
@@ -389,12 +389,12 @@ Feature: user GDPR (General Data Protection Regulation) report
         },
         "groups": {
           "type": "array",
-          "items": [
-            {
-              "type": "string",
-              "pattern": "^%group_id_pattern%$"
-            }
-          ]
+          "minItems": 1,
+          "maxItems": 1,
```

Comment:
for now `minItems` and `maxItems` are same 
it strict because we usually assert exactly given elements in arrays

we'll combine to one `itemsCount` later